modules/integration/src/test/scala/scala/cli/integration/ScalafixTests.scala

```diff
@@ -0,0 +1,59 @@
+package scala.cli.integration
+
+import com.eed3si9n.expecty.Expecty.expect
+
+class ScalafixTests extends ScalaCliSuite {
```

Comment:
It'd be nice to run this suite for different Scala versions.
Refer to how other integration tests are setup, i.e. `CompileTestDefinitions`

Reply:
Thank you. Looked it up and resolved in https://github.com/VirtusLab/scala-cli/pull/2968/commits/61b18f7bb05fd1035d9fdb6d1aded707b3869c3d